velox/docs/develop/aggregate-functions.rst

```diff
@@ -152,13 +152,31 @@ A simple aggregation function is implemented as a class as the following.
     using IntermediateType = Array<Generic<T1>>;
     using OutputType = Array<Generic<T1>>;
 
+    // If UDAF does not require the use of FunctionState, it is necessary
+    // to declare an empty FunctionState struct.
```

Comment:
nit: Define a struct for function-level states. Even if the aggregation function doesn't use function-level states, it is still necessary to define an empty FunctionState struct.